protocol/x/perpetuals/client/cli/query_params.go

```diff
@@ -0,0 +1,32 @@
+package cli
+
+import (
+	"context"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdQueryParams() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-params",
```

Comment:
Nit: it looks like this is conventionally just `params` in Cosmos SDK ([example](https://github.com/dydxprotocol/cosmos-sdk/blob/b95c66dedbd525a1b3d0f6b851d4fd351887fa23/x/slashing/client/cli/query.go#L114))

So the cmd looks like`dydxprotocold query perptuals params`

Up to you on whether to change the other queries, but for params probably makes sense to stick to conventions

Reply:
I actually copied the style in x/bridge - I think I might keep it this way but file a follow-up ticket to migrate to cosmos style.

Reply:
https://linear.app/dydx/issue/CORE-716/change-grpc-getparams-queries-to-remove-get-from-cli-command